components/engine/integration-cli/docker_cli_rmi_test.go

```diff
@@ -152,7 +152,7 @@ func (s *DockerSuite) TestRmiImageIDForceWithRunningContainersAndMultipleTags(c
 
 	out, _, err := dockerCmdWithError("rmi", "-f", imgID)
 	// rmi -f should not delete image with running containers
```

Comment:
perhaps update the comments as well to match the test; might be really confusing otherwise